build_docker.sh

```diff
@@ -21,8 +21,15 @@ set -eu
 
 eval $(./build_dist.sh shellvars)
 
-docker build \
-  --build-arg VERSION_LONG=$VERSION_LONG \
-  --build-arg VERSION_SHORT=$VERSION_SHORT \
-  --build-arg VERSION_GIT_HASH=$VERSION_GIT_HASH \
-  -t tailscale:$VERSION_SHORT -t tailscale:latest .
+go run ./cmd/mkctr \
```

Comment:
go run github.com/tailscale/mkctr

During dev you can `go mod edit -replace github.com/tailscale/mkctr=$HOME/src/github.com/tailscale/mkctr` or whatever.